orchestration/start_monitor.sh

```
@@ -0,0 +1,59 @@
+
+# Check if monitor is already running
+if pgrep -f "agent_monitor.py" > /dev/null; then
+    echo -e "${YELLOW}⚠️  Agent monitor already running${NC}"
+    echo "   To stop: pkill -f agent_monitor.py"
+    echo "   To view logs: tail -f $LOG_DIR/agent_monitor.log"
+    exit 0
+fi
+
+# Start monitor in background
+echo -e "${GREEN}🚀 Starting monitor...${NC}"
+cd "$SCRIPT_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)